web-client/package.json

```diff
@@ -10,9 +10,14 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "@types/react-redux": "^7.1.7",
+    "immer": "^6.0.2",
+    "prop-types": "^15.7.2",
```

Comment:
do we need prop types here?

Reply:
`interface` and `type` from TS should totally replace the need for `propTypes`

Reply:
> do we need prop types here?

I moved prop-types for devDependencies. Nicely pointed out.

> `interface` and `type` from TS should totally replace the need for `propTypes`

I actually had this doubt as well. From what I read, typescript serves as a check at build time. Prop-types works in runtime. There's actually a utility to infer prop-types from typescript that may become very useful. So, for now, I'd say we keep both. If it becomes evidently useless we'll remove it.